src/metabase/pulse.clj

```diff
@@ -324,12 +325,12 @@
         xmin   (apply min xs)
         xmax   (apply max xs)
         xrange (- xmax xmin)
-        xs'    (map #(/ (- % xmin) xrange) xs)
+        xs'    (map #(/ (float (- % xmin)) xrange) xs)
         ys     (map second rows)
         ymin   (apply min ys)
         ymax   (apply max ys)
         yrange (max 1 (- ymax ymin)) ; `(max 1 ...)` so we don't divide by zero
-        ys'    (map #(/ (- % ymin) yrange) ys)
+        ys'    (map #(/ (float (- % ymin)) yrange) ys) ; cast to float to avoid "Non-terminating decimal expansion" errors
```

Comment:
probably better to cast these to `Double` instead of `Float` so we don't lose too much precision